PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkClusterListRefreshableCombo.kt

```diff
@@ -134,7 +134,8 @@ open class SparkClusterListRefreshableCombo: ILogger, Disposable {
                 clustersSelection.comboBox as JComboBox<IClusterDetail>)
 
         // Only getter here since the select setter has a special behavior
-        open val clusterDetailsWithRefresh: Observable<out List<IClusterDetail>> = hdinsightSparkClustersRefreshed
+        open val clusterDetailsWithRefresh: Observable<out List<IClusterDetail>>
+                get() = hdinsightSparkClustersRefreshed
```

Comment:
Why do we need this change here?

Reply:
Since we need to call hdinsightSparkClustersRefreshed() in every getter invoking. Ennnn, looks like `object SparkClustersServices` also need to change.

Reply:
Maybe you can fix them all in this PR :)

Reply:
Fixed.